presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,77 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Add {func}`from_iso8601_timestamp_nanos` function. ({issue}`5048`)
+* Improve performance of queries that use the `DECIMAL` type. ({issue}`4886`)
+* Improve performance of queries involving `IN` with subqueries by extending support for dynamic
+  filtering and dynamic partition pruning. ({issue}`5017`)
+* Improve performance and latency of queries leveraging dynamic partition pruning with dynamic filters. ({issue}`4988`)
+* Improve performance of queries joining tables with missing or incomplete column statistics when
+  the cost-based optimizer is enabled (which is the default). ({issue}`5141`)
```

Comment:
We don't use the phrase "cost-based optimizer" in the documentation (since it's not correct)

> when cost based optimization is enabled

Reply:
I know it is not strictly correct. We used "cost-based optimizer" and "cost based optimizer" in the past, even though the correct term is "cost-based optimizations". -- It's because all the users understand "cost-based optimizer" and this is how they refer to this set of features ("lingua franca")

Reply:
We should just check what is grammatically correct and follows the style guide we use and then just update all the docs in one sweep

Reply:
@findepi, on the other hand, we should break that habit. We shouldn't call the cost-based optimizer as if it were a separate thing. There's only one optimizer in Presto, with the ability to do cost-based optimizations for some plan shapes.

Reply:
> There's only one optimizer in Presto

that's true (assuming we don't count PlanOptimizer subclasses but rather PlanOptimizers class as a whole)

But that is also something users are not interested in.

Anyway, I am OK with the term "cost-based optimizations" (people will still read this as "cost-based optimizer" 😉)